yoti-sdk-api/src/main/java/com/yoti/api/client/DigitalIdentityClient.java

```diff
@@ -81,8 +81,18 @@ public ShareSessionQrCode fetchShareQrCode(String qrCodeId) throws DigitalIdenti
         return identityService.fetchShareQrCode(sdkId, keyPair, qrCodeId);
     }
 
-    public Object fetchShareReceipt(String receiptId) {
-        return identityService.fetchShareReceipt(receiptId);
+    /**
+     * Retrieve the user profile, containing the user's defined attributes, from the share receipt
```

Comment:
I think "user profile" no longer applies. For share v2 there's no "profile".

I think this would be more accurately phrased like so:
```
Retrieve the decrypted share receipt.

<p>A receipt will contain the shared user attributes.</p>
```